templates/partials/tab-background.html

```diff
@@ -75,43 +80,43 @@ <h2>{{localize "WITCHER.background.activeEffect"}}<a class="add-active-effect"><
                     <th></th>
                 </tr>
                 {{#each system.general.lifeEvents as |events ev|}}
-                <tr class="tablerow lifeEvents {{ev}}" data-event="{{ev}}" >
-                    <td class="{{ev}} events">{{ev}}</td>
-                    <td>
-                        <input class="input lifeEvents" name="system.general.lifeEvents.{{ev}}.value" type="text"
-                            value="{{events.value}}"/>
-                    </td>
-                    <td>
-                        {{#unless events.isOpened}}
-                        <a class="life-event-display"><i class="fas fa-chevron-right"></i></a>
-                        {{/unless}}
-                        {{#if events.isOpened}}
-                        <a class="life-event-display"><i class="fas fa-chevron-down"></i></a>
-                        {{/if}}
-                    </td>
-                </tr>
-                {{#if events.isOpened}}
-                <tr>
-                    <td colspan="3">  
-<textarea name="system.general.lifeEvents.{{ev}}.details">{{events.details}}</textarea>
-                    </td>
-                </tr>
-                {{/if}}
+                    <tr class="tablerow lifeEvents {{ev}}" data-event="{{ev}}">
+                        <td class="{{ev}} events">{{ev}}</td>
+                        <td>
+                            <input class="input lifeEvents" name="system.general.lifeEvents.{{ev}}.value" type="text"
+                                   value="{{events.value}}"/>
+                        </td>
+                        <td>
+                            {{#unless events.isOpened}}
+                                <a class="life-event-display"><i class="fas fa-chevron-right"></i></a>
+                            {{/unless}}
+                            {{#if events.isOpened}}
+                                <a class="life-event-display"><i class="fas fa-chevron-down"></i></a>
+                            {{/if}}
+                        </td>
+                    </tr>
+                    {{#if events.isOpened}}
+                        <tr>
+                            <td colspan="3">
+                                <textarea name="system.general.lifeEvents.{{ev}}.details">{{events.details}}</textarea>
```

Comment:
idem, just test it to make sure that there is no issue while saving thistext area